components/image/image.html

```diff
@@ -0,0 +1,141 @@
+                </a>
+                <a href="/components/badge/badge.html">
+                    <li class="component">Badge</li>
+                </a>
+                <a href="/components/button/button.html">
+                    <li class="component">Button</li>
+                </a>
+                <a href="/components/card/card.html">
+                    <li class="component">Card</li>
+                </a>
+                <a href="/components/image/image.html">
+                    <li class="component active">Image</li>
+                </a>
+                <a href="/components/input/input.html">
+                    <li class="component">Input</li>
+                </a>
+                <a href="/components/list/list.html">
+                    <li class="component">List</li>
+                </a>
+                <a href="/components/modal/modal.html">
+                    <li class="component">Modal</li>
+                </a>
+
+            </ul>
+
+        </div>
+
+        <div class="doc-content">
+            <div class="content-wrapper">
+                <div class="content-title "> Image<span class="colored-text">.</span></div>
+                <div class="content-subtitle"> Images play an integral part in making a webiste look visually pleasing & informative.<br> They can be responsive in design & can be modified into different sizes.
+                </div>
+                <div class="divider"></div>
+
+                <div class="content-title2"> Responsive image</div>
+                <div class="content-subtitle">The responsive image expands to cover the parent container.<br> Add <strong>img-responsive</strong> class and resize the browser window to see the responsiveness.
+                </div>
+
+                <div class="example-div">
+                    <img class="img-responsive" src="/assets/beach-repons.png">
+
+
+
+                </div>
+
+                <div class="code-snippet">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=shades-of-purple&wt=none&l=auto&width=672.1749889850616&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=54px&ph=57px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522img-responsive%2522%2520src%253D%2522...%2522%253E%250A%250A"
+                        style="width: 672px; height: 226px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+                <div class="divider"></div>
+
+                <div class="content-title2"> Rounded image</div>
+                <div class="content-subtitle"> Rounded images can used on profile avatars.
+                </div>
+
+                <div class="example-div">
+                    <img class="img-round" src="/assets/bored-ape-minty.png">
+
```

Comment:
can we add alt attribute here for accessibility?

Reply:
Done !